src/components/NavBottom/NavBottom.js

```diff
@@ -1,9 +1,15 @@
 import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import GLOBAL_NAV from './globalNav';
 import './NavBottom.scss';
 
 const NavBottom = () => {
   const [currentId, setCurrentId] = useState();
+  const navigate = useNavigate();
+
+  const moveCategory = id => {
+    navigate(`http://localhost:3000/dog?category=${id}`);
+  }; //네비게이트 url 넘겨주는
```

Comment:
```suggestion
    navigate(`/dog?category=${id}`);
  }; //네비게이트 url 넘겨주는
```
앞에 localhost는 제거해도 될 것 같습니다